services/data/postgres_async_db.py

```diff
@@ -242,6 +242,7 @@ async def execute_sql(self, select_sql: str, values=[], fetch_single=False,
             return aiopg_exception_handling(error), None
         except (Exception, psycopg2.DatabaseError) as error:
             self.db.logger.exception("Exception occured")
+            print(error)
```

Comment:
is this some leftover from testing? Otherwise if you need to print out the exception, the `logger.exception()` should already be taking care of this automatically.

Reply:
Thank you for posting this anyways. This is a good reminder as I may have forgotten. I have removed this :)